src/models/developmentAccount.ts

```diff
@@ -0,0 +1,124 @@
+import { ListMeta, Model, ModelList } from '../model'
+import { Application, ApplicationListResponse, ApplicationResponse, Applications, ApplicationSorting } from '../models/application'
+import { AuthenticatedResponse, SessionHandler } from '../session'
+import { DevelopmentAccountRelationship, DevelopmentAccountRelationshipListResponse, DevelopmentAccountRelationshipResponse, DevelopmentAccountRelationshipRole, DevelopmentAccountRelationships, DevelopmentAccountRelationshipSorting } from './developmentAccountRelationship'
+import { DevelopmentAccountRelationshipRequest, DevelopmentAccountRelationshipRequestListResponse, DevelopmentAccountRelationshipRequestResponse, DevelopmentAccountRelationshipRequests, DevelopmentAccountRelationshipRequestSorting } from './developmentAccountRelationshipRequest'
+
+export enum DevelopmentAccountSorting {
+  ID = 'id',
+  Company = 'company',
+}
+
+export interface DevelopmentAccountData {
+  id: number
+  company: string
+  address: string
+  websiteUrl: string
+}
+
+export interface DevelopmentAccountResponse extends AuthenticatedResponse {
+  developmentAccount: DevelopmentAccountData
+}
+
+export interface DevelopmentAccountListResponse extends AuthenticatedResponse {
+  developmentAccounts: DevelopmentAccountData[]
+  developmentAccountsMeta: DevelopmentAccountListResponseMeta
+}
+
+export interface DevelopmentAccountListResponseMeta extends ListMeta {
+  company?: string
```

Comment:
This meta information is not included in the API's response. You should add it to the API.